.gitignore

```diff
@@ -70,3 +70,4 @@ db.sqlite3
 secret
 node_modules
 webpack-stats.json
+service/out/*
```

Comment:
This line was not committed in my previous PR, so I'm adding it here.  It's unrelated to this issue.